admin/app/model/AdminLifecycle.scala

```diff
@@ -118,10 +110,7 @@ class AdminLifecycle(appLifecycle: ApplicationLifecycle,
     jobs.deschedule("FastlyCloudwatchLoadJob")
     jobs.deschedule("R2PagePressJob")
     jobs.deschedule("AnalyticsSanityCheckJob")
-    jobs.deschedule("FrontPressJob")
     jobs.deschedule("RebuildIndexJob")
-    jobs.deschedule("MatchDayRecorderJob")
-    jobs.deschedule("SentryReportJob")
```

Comment:
I'm assuming the `FrontPressJob` and `SentryReportJob` don't exist any more then